examples/nlp/semantic_similarity_with_bert.py

```diff
@@ -0,0 +1,363 @@
+"""
+Title: Semantic Similarity with BERT
+Author: Mohamad Merchant
+Date created: 2020/08/15
+Last modified: 2020/08/15
+Description: Natural Language Inference by Fine tuning BERT model on SNLI Corpus.
+"""
+"""
+## **Introduction**
+
+Semantic Similarity or Natural Languauge Inference is the task of determining how two
+sentences are similar to each other in terms of their meaning.
+This example demonstrates the use of SNLI (Standford Natural Language Inference) Corpus
+to perform semantic similarity with Transformers.
+We will fine-tune BERT model by providing two sentences as inputs and it outputs the
+probability of the similarity between sentences.
+
+### References
+* [BERT](https://arxiv.org/pdf/1810.04805.pdf)
+* [SNLI](https://nlp.stanford.edu/projects/snli/)
+"""
+
+"""
+## Setup
+"""
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+import transformers
+from transformers import BertTokenizer, TFBertModel
+
+"""
+## Configurations
+"""
+
+MAXLEN = 128
+BATCH_SIZE = 32
+EPOCHS = 4
+LR = 3e-5
+np.random.seed(42)
+# we will use base-base-uncased pretrained model
+MODEL = "bert-base-uncased"
```

Comment:
"MODEL" is too generic, this is "pretrained_model_name"

Reply:
removed "MODEL"  from configurations and added description comment while loading.